demo/twenty_news/build.gradle

```diff
@@ -17,15 +17,10 @@ dependencies {
   compile 'org.slf4j:slf4j-simple:1.7.7'
 }
 
-compileScala {
```

Comment:
Alright, working on this. I assume we are happy with latest Gradle 2.x? Or wanna test out the fresh gradle 3?